src/Purview.Logging.SourceGenerator/Emitters/DependencyInjectionMethodEmitter.cs

```diff
@@ -19,7 +19,7 @@ public DependencyInjectionMethodEmitter(string interfaceName, string className,
 
 	public string Generate()
 	{
-		StringBuilder builder = new();
+		StringBuilder builder = new(Helpers.DefaultStringBuilderCapacity);
```

Comment:
Small improvement not related to benchmarks. It will just slightly improve emit speed of the generated code